client/server/pages/index.js

```diff
@@ -689,6 +690,12 @@ export default function pages() {
 				req.context.sectionGroup = section.group;
 			}
 
+			if ( Array.isArray( section.links ) ) {
+				section.links.forEach( ( link ) =>
+					req.context.store.dispatch( setDocumentHeadLink( link ) )
+				);
+			}
```

Comment:
This looks like a good idea. I wonder if there is another standard way of doing this.

Reply:
I'm not sure either. I'm not a big fan of having the font definition in `section.js`, as it feels a bit too specific for the purpose of this file. But I didn't want to add a condition that checks for the pricing page in a React component.